gvsbuild/projects.py

```diff
@@ -1256,6 +1261,52 @@ def build(self):
     def post_install(self):
         self.exec_cmd(r'%(gtk_dir)s\bin\gdk-pixbuf-query-loaders.exe --update-cache')
 
+@project_add
+class Project_librsvg_rust(Tarball, Project):
```

Comment:
since this is now the default, I would call this `class Project_librsvg` and rename the other to `_old` or `_legacy`